### PR TITLE
Added .github folder to repo

### LINKED ISSUE #166: Let's bring conformity to this repo
[Issue and Pull-Request Templates](https://github.com/blog/2111-issue-and-pull-request-templates)

I'm gonna start on this and run it by y'all in a PR

### PR DESCRIPTION
**Pull request by asclines**
## Description

This PR brings in a new folder, ".github" which is described [here](https://github.com/blog/2111-issue-and-pull-request-templates)
Basically 3 things are done here:
- Move Contributing.md to the .github folder.
- Create an issue template
- Create a Pull-request template (Which this PR is following)

## Related Issue(s)

Closes #166 

## Motivation and Context

As the project grows, communication across contributors is vital to its success.  Hopefully, with a little conformity, we can make it a little easier on everybody. 

## How Has This Been Tested?

Only time will tell on this. 
AKA, testing this will be done by essentially usability testing. <- WE are the users.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Chore (documentation, refactoring, basically not a feature or a bug fix)

## Checklist:

- [ ] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Of course, as always, this is merely **MY** opinion on how this should look. With this PR, I know ask **YOU** what you think. Any suggestions?

### CODE REVIEW

**Comment by asclines** [comment]
@CodeSpaceHQ/clubmengel 🥇

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+2.  
+3.  
+4.  
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
```
Comment (isaac-gs) [inline .github/ISSUE_TEMPLATE.md]: I would suggest taking from Scikit-Learn and using this

>

Comment (isaac-gs) [inline .github/ISSUE_TEMPLATE.md]: I'd suggest doing what Scikit-Learn does, and having people run this,

> Please run the following snippet and paste the output below.
> import platform; print(platform.platform())
> import sys; print("Python", sys.version)
> import numpy; print("NumPy", numpy.__version__)
> import scipy; print("SciPy", scipy.__version__)
> import sklearn; print("Scikit-Learn", sklearn.__version__)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<!--- Why is this change required? What problem does it solve? -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+
+
+## Types of changes
```
Comment (isaac-gs) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: These are great!

**Review by isaac-gs — APPROVED** [review]
Overall really great, just one clarification that we might want to add to make lives easier

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (asclines) [reply .github/ISSUE_TEMPLATE.md]: @ASAAR Where's the source to that? I'd rather just link to that

Comment (isaac-gs) [reply .github/ISSUE_TEMPLATE.md]: https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md#filing-bugs

@asclines sorry about the delay

**Comment by asclines** [comment]
Just FYI, the checks failed because the Travis CI files aren't on this branch yet. That will happen once the #168 is merged into `develop`

**Pull request merged**